CliClient/app/command-tag.js

```diff
@@ -41,7 +50,28 @@ class Command extends BaseCommand {
 		} else if (command == 'list') {
 			if (tag) {
 				let notes = await Tag.notes(tag.id);
-				notes.map((note) => { this.stdout(note.title); });
+                notes.map((note) => {
+                    let line = '';
+                    if (options.long) {
+                        line += BaseModel.shortId(note.id)
+                        line += ' '
+                        line += time.unixMsToLocalDateTime(note.user_updated_time)
```

Comment:
`unixMsToLocalDateTime` should be replaced with `formatMsToLocal`, otherwise the date/time format which is set in the config will be ignored.